src/fheroes2/castle/castle_building_info.cpp

```diff
@@ -188,7 +188,7 @@ namespace
         case BUILD_CASTLE:
             return { 0, 0, 201, 179 };
         case BUILD_CAPTAIN:
-            return { 223, 122, 37, 52 };
+            return { 204, 122, 37, 52 };
```

Comment:
If we move the `x` coordinate to еду left by 19 px we also have to increase the `width` by the same value as the right part of the sprite retains on the same place. :)
```suggestion
            return { 204, 122, 37, 71 };
```

As I checked the changed ROI does not overlap with others, but IMHO it should be different for Town and Castle as for the Castle case.
In this PR a click on the right part of the Castle wall gives the info about the Captain Quarters.
Maybe we have to adjust all ROIs for all Castle buildings in a separate PR.

Reply:
Sure. I will revert this change.